content/docs/operating/storage.md

```diff
@@ -5,10 +5,81 @@ nav_icon: database
 
 # Storage
 
-Prometheus stores its time series data under the directory specified by the flag
-`storage.local.path`. If you suspect problems caused by corruption in the
-database, or you simply want to erase the existing database, you can easily
-start fresh by deleting the contents of this directory:
+Prometheus has a sophisticated local storage subsystem. For indexes,
+it uses [LevelDB](https://github.com/google/leveldb). For the bulk
+sample data, it has its own custom storage layer, which organizes
+sample data in chunks of constant size (1024 bytes payload).
```

Comment:
How about mentioning that each time series is one file: "which organizes sample data in chunks of constant size (...) and stores these chunks in a single file for each time series."